sdk/identity/identity/review/identity.api.md

```diff
@@ -177,6 +206,35 @@ export function getDefaultAzureCredential(): TokenCredential;
 
 export { GetTokenOptions }
 
+// @public
```

Comment:
Hmmm All of this needs documentation. Let’s sync and investigate what alternatives there are, if any.

Reply:
I don't think that we want to export this, ultimately, like I mentioned in the PR comment, this is a work in progress and I don't intend for IdentityClient to be part of the public surface area at the end of the day.